docs/asciidoc/static/include/pluginbody.asciidoc

```diff
@@ -0,0 +1,941 @@
+:branch:          1.5
+:ls_version:		  1.5.0.beta1
+
+= How to write your own Logstash {plugintype} plugin
+
+This HOWTO will walk you through the process of developing a new {plugintype}
+for Logstash. The workflow has changed as of version 1.5 of Logstash, so you'll
+want to follow these new and improved steps to contribute a new plugin!
+
+
```

Comment:
maybe add more info here about how each plugin is self contained and will be its own gem etc..